.github/workflows/cd.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build and upload conda packages
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  conda_deployment:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: '3.9'
+          environment-file: .conda/build_env.yaml
+          auto-update-conda: false
+          auto-activate-base: false
+          show-channel-urls: true
+      - name: Build and upload the conda packages
+        uses: uibcdf/action-build-and-upload-conda-packages@v1.2.0
+        with:
+          python-version: '3.9'
```

Comment:
If creating a `.conda/` directory, it makes more sense to move `meta.yaml` into it and use `meta_yaml_dir: .conda` here.